0_foundation/cloudbuild-destroy.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 Google LLC
```

Comment:
Minor note: instead of copying the same Cloud Build config into each directory, could we use a common one between the different Terraform dirs?

Reply:
For 0_foundation and 1_foundation, the two files are basically the same, but the subsequent destroys have additional stages.

We also want to keep the stages to run independently.  We need to keep the commands simple for developers and users who may not have experience with running automation.